includes/spoolchanges.js

```diff
@@ -71,24 +71,28 @@ module.exports = function(dbUrl, log, bufferSize, callback) {
   c.end();
 
   c.on('response', function(resp) {
-    if (resp.statusCode !== 200) {
-      c.abort();
-      callback(new error.BackupError('SpoolChangesError', `ERROR: Changes request failed with status code ${resp.statusCode}`));
-    } else {
-      console.error('Streaming changes to disk:');
-    }
-  }).pipe(liner())
-    .pipe(change(onChange))
-    .on('finish', function() {
-      processBuffer(true);
-      console.error('');
-      if (!lastSeq) {
-        logStream.end();
-        callback(new error.BackupError('SpoolChangesError', `ERROR: Changes request terminated before last_seq was sent`));
+    request.isFatal(resp, function(err) {
+      if (err) {
+        c.abort();
+        callback(err);
       } else {
-        logStream.write(':changes_complete ' + lastSeq + '\n');
-        logStream.end();
-        callback();
+        console.error('Streaming changes to disk:');
+        resp
+          .pipe(liner())
+          .pipe(change(onChange))
+          .on('finish', function() {
+            processBuffer(true);
+            console.error('');
```

Comment:
Erm, no idea, I just left it in from before. I'll see how bad it looks without or add the `\n`.